modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -3,56 +3,202 @@
+:openid-app-github-url: https://github.com/owncloud/openidconnect
+:mod_rewrite-docs-url: https://httpd.apache.org/docs/current/mod/mod_rewrite.html
+:oc-composer-playground-repo-url: https://github.com/owncloud-docker/compose-playground/tree/master/kopano/konnect
+:examplesdir: ../../../../examples/
 
 == Introduction
 
 "OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
 OpenID Connect allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. The specification suite is extensible, allowing participants to use optional features such as encryption of identity data, discovery of OpenID Providers, and session management, when it makes sense for them."
 -- {openid-connect-url}, "What is OpenID Connect?"
 
+//ownCloud-specific OIDC description and benefits
+== OIDC Benefits
+
+== OIDC App Marketplace Link
+
+// Not currently available
+* https://marketplace.owncloud.com/apps/openidconnect
+
 == Prerequisites
 
-A distributed Memcache setup is required to operate this app - such as Redis or Memcached properly.
-Please follow xref:configuration/server/caching_configuration.adoc[the documentation] on how to set up caching.
+The OpenID Connect app has several mandatory requirements, these are:
+
+* A minimum PHP version of {minimum-php-version}, with the CURL and JSON extensions.
+* xref:configuration/server/caching_configuration.adoc[A distributed memory cache setup], such as _Redis_ or _Memcached_.
+This is because information about tokens need to be cached to prevent round trips to the identity provider. 
+Additionally, in clustered setups this information needs to be available for all nodes, which necessitates a distributed cache.
++
+NOTE: If a memory cache is not available, then you will be prevented from using the ownCloud UI, and see the following message: "_A real distributed mem cache setup is required_" in the ownCloud UI.
+
+== Limitations
+
+=== LDAP Users
+
+LDAP users can log in via OIDC.
+However, this only works with users from LDAP that are available in the ownCloud database, as well as in the Identity Provider.
+For this to work, xref:configuration/server/occ_command.adoc#syncing-user-accounts[user sync] is mandatory.
+
+=== Local/Guest Users
+
+A solution to authenticate local/guest users via OIDC will be available soon.
+
+== Installation
+
+=== Configure ownCloud
+
+The first thing to do is to install {openid-app-marketplace-url}[the OIDC app].
+After it is installed, add the configuration below in either `config/config.php` or a custom config.php file.
 
-== Code Flow
+[source,php]
+----
+include::{examplesdir}configuration/user/oidc/config-with-service-discovery.php[]
```

Comment:
This file does not seem to exist yet.

Reply:
I'll find it and add it in.

Reply:
I was sure that those files were in the repo, but I've been unable to find them anywhere in the repo history. I'll recreate them and add them today.

Reply:
Just found the relevant information and will have it available shortly.